src/main/java/Main.java

```diff
@@ -0,0 +1,17 @@
+import dao.FruitDao;
+import dao.FruitDaoImpl;
+import service.FruitShopService;
+import service.impl.FruitShopServiceImpl;
+
+/**
+ * Feel free to remove this class and create your own.
+ */
+public class Main {
+    public static void main(String[] args) {
+        FruitDao fruitDao = new FruitDaoImpl();
+        FruitShopService fruitShopService = new FruitShopServiceImpl();
+
+        fruitShopService.getResult("inputDateFile.csv", "file.csv");
+        System.out.println(fruitDao.getMap().toString());
```

Comment:
```suggestion
        System.out.println(fruitDao.getMap());
```
toString will be called under the hood